frontend/src/components/common/progress-details/ProgressDetails.tsx

```diff
@@ -201,6 +200,16 @@ const ProgressDetails = ({
           </tr>
         </tbody>
       </table>
+
+      <div className={styles.statsLastUpdated}>
```

Comment:
Could we only show this button when the campaign is complete, since we poll the stats while the campaign is in progress?